### DISCUSSION TITLE
Secondary-purchases trigger fixed-time auction

### DISCUSSION THREAD

**Discussion by will-holley** [RadicalxChange/salsa-nft#4] [Unsolved Questions]
- Gives current owner an opportunity to keep their asset

**Comment by gravenp** [comment]
[This link ](https://ethresear.ch/t/highlight-robin-hansons-more-owner-forgiving-modified-harberger-tax/5720) hits on two different ideas for ways to make PCO more forgiving. 

Definitely a worthwhile area of exploration because the first reaction most people have to PCO is fear of losing their asset without warning. In some use cases (think large CapEx), the asset is only valuable if you have high confidence that you won't be bought out in a given period of time. (Yes, a properly set turnover/tax rate should address some of this too.)

**Comment by gravenp** [comment]
*12/6 Meeting Notes* 

Adding these notes under this open category, because we mostly talked about periodic auctions for PCO assets which is another approach to softening some of the sharp edges of a continuous auction market (@will-holley started this thread with an alternative approach).

The three main motivations that we discussed for these sort of design changes are as follows:

1. Lessen anxiety/fear for having an asset force transferred (big adoption hurdle for most)
2. Some use cases require long-term investment and/or would be completely undercut if the licensor can't get reasonable confidence (or a guarantee) that they'll control the asset long enough (not forever) to reap benefits from their investment 
3. Allow time for licensors and potential bidders to incorporate new information to their valuations

Moving to periodic auctions does come with tradeoffs. The longer the period, the more it looks like perpetual ownership. As Matt pointed out, it is probably incorporate the ideal/natural turnover rate into the determination on period length. They shouldn't be in a 1:1 ratio, but have multiple periods within the average turnover timeframe.

We also discussed Depreciating Licenses (https://anthonyleezhang.github.io/pdfs/dl.pdf) as an alternative implementation of Partial Common Ownership. Depreciating licenses use the periodic auction approach. One of the biggest differences between the implementation proposed by Weyl & Zhang here and what our discussions have focused on is the use of a second-price auction rather than self-assessed values for price setting. In some use cases, there might not be much or any competition for asset to start. Should a second-price auction with only one bid set the price to a minimum value or free? 

In relation to that question on the Geo Web, we look at PCO first and foremost as an efficient allocation mechanism and not primarily a "revenue" generation mechanism (even if we do believe that funding public goods on our network is positive-sum). It might be appropriate for us to have the rate at which someone pays for a license = 0 (excluding transaction fees) if there isn't any competition for it.
